pkg/compression/compression.go

```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/ulikunitz/xz"
 )
 
+const (
+	// GzipCompression indicates gzip compression.
+	GzipCompression = "gzip"
```

Comment:
`compression.GzipCompression` seems unnecessarily redundant.

Reply:
Agreed. Will update with the next push.